rules/community/cloudtrail/cloudtrail_security_group_ingress_anywhere.py

```diff
@@ -0,0 +1,33 @@
+"""Alert on AWS Security Groups that allow ingress from anywhere."""
+from helpers.base import select_key
+from stream_alert.rule_processor.rules_engine import StreamRules
+
+rule = StreamRules.rule
+
+@rule(logs=['cloudwatch:events'],
+      matchers=[],
+      outputs=['aws-s3:sample-bucket',
+               'pagerduty:sample-integration',
+               'slack:sample-channel'],
+      req_subkeys={'detail': ['eventName', 'requestParameters']})
+def cloudtrail_security_group_ingress_anywhere(rec):
+    """
+    author:         @mimeframe
```

Comment:
You got your own name wrong 🤔 @mime-frame

Reply:
I use my twitter handle vs. github :)